tests/phpunit/Writer/CommonTest.php

```diff
@@ -191,13 +194,13 @@ public function testGenerateTmpName()
         $this->assertLessThanOrEqual(64, mb_strlen($tmpName));
     }
 
-    public function testValidateTable()
+    public function testValidateTable(): void
     {
         $table = $this->config['parameters']['tables'][0];
         $this->writer->create($table);
 
-        $res = $this->writer->validateTable($table);
+        $this->writer->validateTable($table);
 
-        $this->assertTrue($res);
+        $this->assertTrue(true);
```

Comment:
To nevypadá na moc kvalitní test :)

Reply:
to mělo testovat nějakou exception? nebo o co jde?

Reply:
Jo už to vidím ona se zároven měnila i funkčnost a nově to nevrací nic místo `bool`. Takže asi ok.